app/src/main/java/com/twilio/video/app/ui/room/ParticipantControllerExtensions.kt

```diff
@@ -0,0 +1,19 @@
+package com.twilio.video.app.ui.room
+
+import com.twilio.video.app.participant.ParticipantViewState
+
+internal fun ParticipantController.updateThumbnails(participants: List<ParticipantViewState>?) {
+    participants?.let {
+        // Add or update any participants in the thumbs list
+        participants.forEach { participant ->
+            if (getThumb(participant.sid) == null) addThumb(participant) else updateThumb(participant)
```

Comment:
Does a one line `if`/`else` follow our style guides?

Reply:
Hmm not sure. We do use spotless to do auto formatting but it doesn't seem to format these. I'll fix this one and try to avoid these one liners moving forward.